README.md

```diff
@@ -50,6 +50,35 @@ To use the blueprint, run the below command:
 jhipster --blueprints generator-jhipster-svelte
 ```
 
+## Using Docker
+
+Download the Dockerfile:
+
+```bash
+mkdir docker
+cd docker
+wget https://github.com/jhipster/generator-jhipster-svelte/raw/master/docker/Dockerfile
+```
+
+Build the Docker images:
+
+```bash
+docker build -t jhipster-generator-svelte:latest .
```

Comment:
instead of `jhipster-generator-svelte`, would like to have short image name

Reply:
pls suggest some other descriptive name , if you have any.